charts/nd-common/templates/_datadog.tpl

```diff
@@ -35,9 +35,9 @@ we add default values to it.
 ad.datadoghq.com/{{ include "nd-common.containerName" . }}.logs: |-
   [
     {
-      "source": {{- default .Chart.Name .Values.datadog.scrapeLogs.source }},
-      "service": {{- default .Chart.Name .Values.datadog.service  }},
-      "log_processing_rules": {{- tpl (toJson .Values.datadog.scrapeLogs.processingRules) $ }}
+      "source": {{ default .Chart.Name .Values.datadog.scrapeLogs.source | toJson }},
```

Comment:
Why?

Reply:
@diranged https://docs.datadoghq.com/agent/logs/advanced_log_collection/?tab=kubernetes

Reply:
This was the output generated by previous code 

```
 annotations:
    ad.datadoghq.com/gophoto.logs: |-
      [
        {
          "source":go,
          "service":gophoto,
          "log_processing_rules":[]
        }
      ]
```

Reply:
I tested in staging by adding quote to values and I was able to see logs in datadog.

Reply:
ok... the `toJson` though is a bit wrong because its not _json_ right there, you just wanted a quoted value.. Can you follow up with:

```yaml
"source": {{ default .Chart.Name .Values.datadog.scrapeLogs.source | quote }},
```